handlers/secrets.go

```diff
@@ -75,3 +81,86 @@ func makeSecretsArray(c *client.Client, secretNames []string) ([]*swarm.SecretRe
 
 	return values, nil
 }
+
+func secretGetHandler(c *client.Client, w http.ResponseWriter, r *http.Request) {
+	secrets, err := c.SecretList(context.Background(), types.SecretListOptions{})
+
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+
+		return
+	}
+
+	var results []requests.Secret
+
+	for _, s := range secrets {
+		results = append(results, requests.Secret{Name: s.Spec.Name})
+	}
+
+	bs, marshalErr := json.Marshal(results)
```

Comment:
instead of short names, it would be nice to have a semantic name like `payload`, `responseBody`, or `resultJSON` instead of `bs` here.

Reply:
I agree.. the pull request template says: "this code follows the style of the project" and the comment above would align with that sentiment. I also mentioned `bs` is not ideal (bs can actually mean something we really don't want to have there)

Reply:
changed